src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -125,7 +125,7 @@ struct MultiPlayerClientGroup : public UI::Box {
 		const UserSettings& user_setting = settings.users.at(id_);
 
 		slot_dropdown_.clear();
-		for (PlayerSlot slot = 0; slot < settings.players.size(); ++slot) {
```

Comment:
You'll want these changes for PlayerSlot and TeamNumber reverted, like I did with DescriptionIndex?

Reply:
yes please